SLHCUpgradeSimulations/Geometry/test/phase2_digi_reco_pixelntuple_cfg.py

```diff
@@ -101,10 +105,9 @@
 # Path and EndPath definitions
 process.digitisation_step = cms.Path(process.pdigi_valid)
 process.L1simulation_step = cms.Path(process.SimL1Emulator)
-process.L1TrackTrigger_step = cms.Path(process.L1TrackTrigger)
+#process.L1TrackTrigger_step = cms.Path(process.L1TrackTrigger) #needs to be re-inserted
```

Comment:
I think that by dropping the offending branch from the input source, now you can put back the L1TT.

Reply:
Agree. Put it back.